### PR TITLE
cmp and zero for pyflatsurf vectors

### PR DESCRIPTION
**Pull request by videlec**

### CODE REVIEW

**Review by saraedum — APPROVED** [review]

File: doc/news/vector-convenience-functions.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**Added:**
+
+* Comparison of vectors in pyflatsurf and implementation of the zero() method
```
Comment (saraedum) [inline doc/news/vector-convenience-functions.rst:3]:
```suggestion
* Added comparison of vectors in pyflatsurf and implementation of the zero() method
```

**Review by saraedum — APPROVED** [review]

**Comment by saraedum** [comment]
Two jobs failed. Probably network errors.

**Comment by videlec** [comment]
@saraedum Could you have a look at the failure in the conda-build test ?

**Comment by saraedum** [comment]
The error will be fixed by https://github.com/conda-forge/fmt-feedstock/pull/28.

**Pull request merged**